### ISSUE TITLE
ag-grid 10.0.0 - Missing interfaces/classes

### ISSUE THREAD

**Issue by dloprodu** [ceolter/ag-grid#1652]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

I'm using ag-grid in a Angular2 project. Before upgrade to the version 10.0.0 the following import worked fine:

`
import {
    IDatasource, 
    IGetRowsParams,
    NavigateToNextCellParams 
} from 'ag-grid/main';
`

But now, this interfaces/clasess are missing.

**Expected behavior**

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **ag-Grid version:** 10.0.0

* **Browser:**

* **Language:** [TypeScript]

**Comment by sbublava** [comment]
Workaround:

import { IDatasource, IGetRowsParams } from 'ag-grid/dist/lib/rowModels/iDatasource';

But this is indeed a bit annoying ...

**Comment by seanlandsman** [comment]
Hi,

This issue has now been fixed - try upgrade to 10.0.1

thanks
